changes/ocd-4288.sql

```diff
@@ -0,0 +1,131 @@
+alter table openchpl.test_tool drop column if exists name;
+
+alter table openchpl.test_tool drop column if exists description;
+
+alter table openchpl.test_tool drop column if exists retired;
+
+--------------------------------------------------------------------------
+
+alter table openchpl.functionality_tested add column if not exists value text;
+
+alter table openchpl.functionality_tested add column if not exists regulatory_text_citation text;
+
+alter table openchpl.functionality_tested add column if not exists additional_information text;
+
+alter table openchpl.functionality_tested add column if not exists start_day date;
+
+alter table openchpl.functionality_tested add column if not exists end_day date;
+
+alter table openchpl.functionality_tested add column if not exists required_day date;
```

Comment:
Do we need this column for functionality tested? I see that we had it but removed it for test tools.

Reply:
It is required for Functionality Tested.